tests/tests/tier0/bluechi-agent-set-loglevel/test_bluechi_agent_set_loglevel.py

```diff
@@ -0,0 +1,51 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+from typing import Dict
+from bluechi_test.test import BluechiTest
+from bluechi_test.container import BluechiControllerContainer, BluechiNodeContainer
+from bluechi_test.config import BluechiControllerConfig, BluechiNodeConfig
+
+NODE_FOO = "node-foo"
+
+
+def get_str_to_set_loglevel(node, loglevel):
+    return f"bluechictl set-loglevel {node} {loglevel}"
+
+
+def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
```

Comment:
I'd prefer to change order of steps (assuming that default log level is INFO:

1. Stop agent service
2. Save current time to use as a boundary for searching journal
3. Start agent service
4. Check that journal since time from step 2 doesn't contain any DEBUG message from bluechi-agent
5. Change log level to DEBUG
6. Execute some action for agent which should emit some debug message
7. Check that journal since time from step 2 contains at least one DEBUG message from bluechi-agent